### PR TITLE
Kubespray 2.20 custom commits

### PR DESCRIPTION
**Pull request by Moumoutaru**
This merge requests aims to pull all of the custom commits we've made to Kubespray to get it to work for us. All of these commits have been cherry picked to `kubespray-2.20-custom-commits` branch. The commits to be cherry picked were generated via this command:

```
git checkout checkout release-2.18-power-1
git log --reverse --since='Nov 21, 2022' --pretty=%H > k8s-custom-commits
```

### CODE REVIEW

File: roles/kubernetes/node/tasks/main.yml [hunk 1]
```diff
@@ -87,6 +87,7 @@
   changed_when: false
   check_mode: no
   register: sysctl_bridge_nf_call_iptables
+  check_mode: false
```
Comment (Moumoutaru) [inline roles/kubernetes/node/tasks/main.yml]: This needs fixed -- check mode is defined [two lines above ](https://github.com/powerhome/kubespray/pull/13/files#diff-d8573eb6ba49a64e94117e08832eabf1319b675724997469cf0bea039bb0a331R88)

File: galaxy.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
+namespace: powerhome
+description: Deploy a production ready Kubernetes cluster
+name: kubespray
+version: 2.18.1
```
Comment (Moumoutaru) [inline galaxy.yml]: bump version to 2.20.X?

**Comment by rob-crowston** [comment]
I ran through this line-by-line multiple times yesterday, comparing with the [original changes](https://github.com/powerhome/kubespray/pull/1/files) and I can't see anything that is missing. I noticed a couple of calico changes but when looking at the branch I could see those tasks so I think we're good here but it may be worth getting someone who is familiar with those original changes to give this a once over.

File: roles/kubernetes/node/tasks/main.yml (hunk 1 above)
Comment (Moumoutaru) [reply roles/kubernetes/node/tasks/main.yml]: Remove `check_mode: false`

**Review by rob-crowston — APPROVED** [review]

**Pull request merged**